addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -362,7 +362,7 @@ export const editorCommands = {
         getDeepRange(editor.editable, { splitText: true, select: true, correctTripleClick: true });
         const isAlreadyBold = getSelectedNodes(editor.editable)
             .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue.trim().length)
-            .find(n => isBold(n.parentElement));
+            .every(n => isBold(n.parentElement));
```

Comment:
I suggest adding some format tests.

Reply:
Good idea. I'll wait for https://github.com/odoo/odoo/pull/85408 to be merged then maybe since it adds a file for formats testing... Then I'll adapt this branch to add the new tests.

Reply:
I did it there directly finally: https://github.com/odoo/odoo/pull/85408.